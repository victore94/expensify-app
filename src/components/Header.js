import React from 'react'
import { NavLink } from 'react-router-dom'

const Header = () => (
    <header>
        <h1>Expensify</h1>
        <NavLink exact to='/' activeClassName='is-active'>Dashboard</NavLink>
        <NavLink to='/create' activeClassName='is-active'>Go to Create</NavLink>
    </header>
)

export default Header;