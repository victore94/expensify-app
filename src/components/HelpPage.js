import React from 'react'

const HelpPage = () => (
    <div>
        this is my helpPage component
    </div>
)

export default HelpPage;