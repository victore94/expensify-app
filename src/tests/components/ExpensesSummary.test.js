import React from 'react'
import { shallow } from 'enzyme'
import { ExpensesSummary } from '../../components/ExpensesSummary';

test('correctly render expense summary with 1 expense ', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={235} />)
    expect(wrapper).toMatchSnapshot()
})

test('correctly render expense summary with multiple expensed ', () => {
    const wrapper = shallow(<ExpensesSummary expenseCount={23} expensesTotal={2352134987} />)
    expect(wrapper).toMatchSnapshot()
})