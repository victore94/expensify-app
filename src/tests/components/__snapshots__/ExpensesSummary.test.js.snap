// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`correctly render expense summary with 1 expense  1`] = `
<div>
  <h1>
    Viewing 
    1
     
    expense
     totalling 
    $2.35
  </h1>
</div>
`;

exports[`correctly render expense summary with multiple expensed  1`] = `
<div>
  <h1>
    Viewing 
    23
     
    expenses
     totalling 
    $23,521,349.87
  </h1>
</div>
`;
